src/modules/map/components/CompareSlider.vue

```diff
@@ -53,6 +86,7 @@ export default {
                 event.type === 'touchmove' ? event.touches[0].pageX : event.pageX
             // THIS DIES ON CLIENT SMALLER THAN SCREEN
```

Comment:
what does that mean? shouldn't we address this?

Reply:
I'm aware of it and I'll work on correcting it before merging. Here is a description of the problem :

When dragging and dropping the component, it takes the position of the cursor compared to the screen rather than to the page. So if your web browser is not occupying the whole screen, it works, but your cursor position is not where the component will end.

I've tried with event.pageX, screen and client and in each case it takes the screen as a whole rather than the width of the browser.

Reply:
just for information : this has been solved